src/webhooks/interfaces/webhook-directory.interface.ts

```diff
@@ -1,10 +1,25 @@
-import { Directory } from '../../directory-sync/interfaces';
+export type WebhookDirectoryState =
+  | 'active'
+  | 'validating'
+  | 'invalid_credentials'
+  | 'inactive'
+  | 'deleting';
 
-export interface WebhookDirectory extends Omit<Directory, 'state'> {
-  state:
-    | 'active'
-    | 'validating'
-    | 'invalid_credentials'
-    | 'inactive'
-    | 'deleting';
+interface WebhookDirectoryDomain {
+  object: 'organization_domain';
+  id: string;
+  domain: string;
+}
+
+export interface WebhookDirectory {
+  object: 'directory';
+  id: string;
+  external_key: string;
+  type: string;
```

Comment:
Should this be a `DirectoryType`?

```suggestion
  type: DirectoryType;
```

Reply:
Ah, nevermind, I see we don't currently have a `DirectoryType` in the SDK.

Reply:
@maxdeviant pushed the change